docs/jah/ack/index.md

```diff
@@ -0,0 +1,123 @@
+---
+---
+
+# Ack
+
+Simple sample player
+
+## Features
+
+- 8 voice sample playback of mono or stereo samples
+- Sample start / end and loop positions
+- Resonant multimode filter
+- Envelopes for volume and filter cutoff
+- Master delay and reverb
+
+## Commands
+
+_Note: channels are indexed 0-7 in commands._
+
+### Triggering
+
+- trig i: triggers single channel given a channelnum.
+- multiTrig iiiiiiii: triggers multiple channels. one 1/0 argument for each channel. 1 means trigger.
+
+### Channel Settings
+
+First argument is always channelnum.
```

Comment:
nit (feel free to ignore!):

some settings descriptions end with a `.` and some don't.  if you're finicky, maybe make them consistent?

Reply:
thanks, hope to revise these a bit later today